test/ffmpeg-vaapi/vpp/brightness.py

```diff
@@ -8,40 +8,58 @@
 from ..util import *
 
 spec = load_test_spec("vpp", "brightness")
+spec_r2r = load_test_spec("vpp", "brightness", "r2r")
 
 _NOOP_LEVEL = 50 # i.e. 0.0 in ffmpeg range
 
-@slash.requires(have_ffmpeg)
-@slash.requires(have_ffmpeg_vaapi_accel)
-@slash.requires(*have_ffmpeg_filter("procamp_vaapi"))
-@slash.parametrize(*gen_vpp_brightness_parameters(spec))
-@platform_tags(VPP_PLATFORMS)
-def test_default(case, level):
-  params = spec[case].copy()
-  params.update(
+def init(tspec, case, level):
+  tparams = tspec[case].copy()
+  tparams.update(
     level = level, mlevel = mapRange(level, [0, 100], [-100.0, 100.0]),
-    mformat = mapformat(params["format"]))
-  params["ofile"] = get_media()._test_artifact(
-    "{}_brightness_{level}_{format}_{width}x{height}"
-    ".yuv".format(case, **params))
+    mformat = mapformat(tparams["format"]))
+
+  if tparams["mformat"] is None:
+    slash.skip_test("{format} format not supported".format(**tparams))
 
-  if params["mformat"] is None:
-    slash.skip_test("{format} format not supported".format(**params))
+  return tparams
 
+def call_ffmpeg(params):
   call(
     "ffmpeg -hwaccel vaapi -vaapi_device /dev/dri/renderD128 -v debug"
     " -f rawvideo -pix_fmt {mformat} -s:v {width}x{height} -i {source}"
     " -vf 'format=nv12,hwupload,procamp_vaapi=b={mlevel}"
     ",hwdownload,format=nv12'"
     " -pix_fmt {mformat} -vframes {frames} -y {ofile}".format(**params))
 
+def gen_output(case, params):
+  name = "{case}_brightness_{level}_{format}_{width}x{height}".format(case = case, **params)
+
+  if params.get("r2r", None) is not None:
+     name =  name + "_r2r"
```

Comment:
please fix indentation (there's an extra space here and in other gen_output's)

Reply:
Also, how about `name += ...` instead?

Reply:
thanks, fixed extra space issue and used name +=